modules/rosa-hcp-create-network.adoc

```diff
@@ -0,0 +1,132 @@
+    Default: '10.0.0.0/16'
+----
++
+The command takes about 5 minutes to run and provides regular status updates from AWS as resources are created. If there is an issue with CloudFormation, a rollback is attempted. For all other errors are encountered, please follow the error message instructions or contact AWS support.
+
+. When completed, you receive a summary of the created resources:
++
+[source,bash]
+----
+INFO[0140] Resources created in stack:                  
+INFO[0140] Resource: AttachGateway, Type: AWS::EC2::VPCGatewayAttachment, ID: <gateway-id> 
+INFO[0140] Resource: EC2VPCEndpoint, Type: AWS::EC2::VPCEndpoint, ID: <vpce-id> 
+INFO[0140] Resource: EcrApiVPCEndpoint, Type: AWS::EC2::VPCEndpoint, ID: <vpce-id>
+INFO[0140] Resource: EcrDkrVPCEndpoint, Type: AWS::EC2::VPCEndpoint, ID: <vpce-id> 
+INFO[0140] Resource: ElasticIP1, Type: AWS::EC2::EIP, ID: <IP>
+INFO[0140] Resource: ElasticIP2, Type: AWS::EC2::EIP, ID: <IP> 
+INFO[0140] Resource: InternetGateway, Type: AWS::EC2::InternetGateway, ID: igw-016e1a71b9812464e 
+INFO[0140] Resource: KMSVPCEndpoint, Type: AWS::EC2::VPCEndpoint, ID: <vpce-id> 
+INFO[0140] Resource: NATGateway1, Type: AWS::EC2::NatGateway, ID: <nat-gateway-id> 
+INFO[0140] Resource: PrivateRoute, Type: AWS::EC2::Route, ID: <route-id> 
+INFO[0140] Resource: PrivateRouteTable, Type: AWS::EC2::RouteTable, ID: <route-id> 
+INFO[0140] Resource: PrivateSubnetRouteTableAssociation1, Type: AWS::EC2::SubnetRouteTableAssociation, ID: <route-id>
+INFO[0140] Resource: PublicRoute, Type: AWS::EC2::Route, ID: <route-id> 
+INFO[0140] Resource: PublicRouteTable, Type: AWS::EC2::RouteTable, ID: <route-id> 
+INFO[0140] Resource: PublicSubnetRouteTableAssociation1, Type: AWS::EC2::SubnetRouteTableAssociation, ID: <route-id> 
+INFO[0140] Resource: S3VPCEndpoint, Type: AWS::EC2::VPCEndpoint, ID: <vpce-id> 
+INFO[0140] Resource: STSVPCEndpoint, Type: AWS::EC2::VPCEndpoint, ID: <vpce-id> 
+INFO[0140] Resource: SecurityGroup, Type: AWS::EC2::SecurityGroup, ID: <security-group-id> 
+INFO[0140] Resource: SubnetPrivate1, Type: AWS::EC2::Subnet, ID: <subnet-id-1> \ <1>
+INFO[0140] Resource: SubnetPublic1, Type: AWS::EC2::Subnet, ID: <subnet-id-1> \ <1>
+INFO[0140] Resource: VPC, Type: AWS::EC2::VPC, ID: <vpc-id>
+INFO[0140] Stack rosa-network-stack-5555 created \ <2>
+----
++
+--
+<1> These two subnet IDs are used to create your cluster when using the `rosa create cluster` command.
+<2> The network stack name is used to delete the resource later.
++
+[NOTE]
+====
+Optional: You can add the created network stack to your compute security IDs by selecting the network stack when prompted as follows:
+[source,terminal]
+----
+? Additional 'Compute' Security Group IDs (optional):  [Use arrows to move, space to select, <right> to all, <left> to none, type to filter, ? for more help]
+> [ ]  sg-<security-group-id> ('rosa-network-stack-5555-SecurityGroup-aaa')
+----
+====
+--
+
+. Delete the CloudFormation stack by running the following command:
```

Comment:
Is this part of creating the network, or is this how you would get rid of a network you had previously created? Should it move to a separate module?